.env.example

```diff
@@ -1,2 +1,2 @@
 VOCDONI_ENVIRONMENT=prod
-FEATURES='{"faucet":true,"vote":{"anonymous":true,"overwrite":true,"secret":true},"login":["web3","web2"],"census":["spreadsheet","token","web3"],"languages":["en","es","ca"]}'
+FEATURES='{"faucet":true,"vote":{"anonymous":true,"overwrite":true,"secret":true,"customization":true},"types":{"single":true,"multi":true,"approval":true,"participatory":true,"borda":true},"login":["web3","web2"],"census":["spreadsheet","token","web3"],"unimplemented_census":["phone","email","crm","database","digital_certificate"],"languages":["en","es","ca"]}'
```

Comment:
This example does not match with the actual implementation.

Reply:
@gerouvi I still miss the `unimplemented_voting_types`, `unimplemented_census` and `voting_types` in that FEATURES env 🤔